scripts/prepare_release.sh

```diff
@@ -65,27 +65,33 @@ git pull
 git checkout -b ${VERSION}-${SUFFIX}
 
 
-if [[ "${SUFFIX}" = "bom" ]]; then
-  cd boms/cloud-oss-bom
-fi
-
-# Updates the pom.xml with the version to release.
-mvn versions:set versions:commit -DnewVersion=${VERSION} -DgenerateBackupPoms=false
-
-# Tags a new commit for this release.
-git commit -am "preparing release ${VERSION}-${SUFFIX}"
-git tag v${VERSION}-${SUFFIX}
-
 # Updates the pom.xml with the next snapshot version.
 # For example, when releasing 1.5.7, the next snapshot version would be 1.5.8-SNAPSHOT.
 NEXT_SNAPSHOT=${NEXT_VERSION}
 if [[ "${NEXT_SNAPSHOT}" != *-SNAPSHOT ]]; then
   NEXT_SNAPSHOT=${NEXT_SNAPSHOT}-SNAPSHOT
 fi
-mvn versions:set versions:commit -DnewVersion=${NEXT_SNAPSHOT} -DgenerateBackupPoms=false
 
-# Commits this next snapshot version.
-git commit -am "${NEXT_SNAPSHOT}"
+if [[ "${SUFFIX}" = "bom" ]]; then
+  cd boms/cloud-oss-bom
+fi
+
+if [[ "${SUFFIX}" = "gradle" ]]; then
+  cd gradle-plugin
+  # Changes the version for release and creates the commits/tags.
+  echo | ./gradlew release -Prelease.releaseVersion=${VERSION} \
+      -Prelease.newVersion=${NEXT_SNAPSHOT}
+else
+  # Updates the pom.xml with the version to release.
+  mvn versions:set versions:commit -DnewVersion=${VERSION} -DgenerateBackupPoms=false
+  # Tags a new commit for this release.
+  git commit -am "preparing release ${VERSION}-${SUFFIX}"
+  git tag v${VERSION}-${SUFFIX}
+  mvn versions:set versions:commit -DnewVersion=${NEXT_SNAPSHOT} -DgenerateBackupPoms=false
+
+  # Commits this next snapshot version.
+  git commit -am "${NEXT_SNAPSHOT}"
+fi
```

Comment:
For "bom" and "dependencies", this script uses "mvn versions" to set version.

For "gradle", this script uses net.researchgate.release plugin.